tests/languages/false/assembler_code_feature.test

```diff
@@ -0,0 +1,19 @@
+0`
+65535`
+
+----------------------------------------------------
+
+[
```

Comment:
This isn't a nit but a tip for the future:

This JSON looks like you formatted it yourself. With the command `npm run test:languages -- --accept`, the test suite [will automatically insert](https://prismjs.com/test-suite.html#writing-tests-writing-your-test) the formatted actual token stream for you.

Reply:
Ah, I unreasonably forgot about `--accept`. I’ve never used the feature, because I write tests before I write regexes. I should and I will use it for “post-processing”.

Reply:
Thank you for reminding!